### ISSUE TITLE
Configuring TLS

### ISSUE THREAD

**Issue by tony-gamble-asos** [microsoft/service-fabric-traefik#8]
According to your documentation :
```
traefik.http.[endpointName].router.tls.options Enable TLS on the route ['true'/'false']. 
```

This results in the following router set up

```
    fabric-Application2-Web1-http-defaultEP-1d6dda1e-6a45-4bb9-bbd9-2ed7c11a4fe9:
      entryPoints:
      - websecure
      middlewares:
      - sf-stripprefixregex_nonpartitioned
      rule: PathPrefix(`/WeatherForecast`)
      service: fabric-Application2-Web1-http-defaultEP-1d6dda1e-6a45-4bb9-bbd9-2ed7c11a4fe9
      tls:
        options: "true"
```

The intended use of options is to provide a reference to defined options, rather than a Boolean value

https://doc.traefik.io/traefik/routing/routers/#tls

Also there is no way to configure TLS with no options specified ie 

```
http:
  routers:
    Router-1:
      rule: "Host(`foo-domain`) && Path(`/foo-path/`)"
      service: service-id
      # will terminate the TLS request
      tls: {}
```

**Comment by egaribay77** [comment]
This should now be fixed with v1.0.0 release (https://github.com/microsoft/service-fabric-traefik/pull/17).

**Comment by egaribay77** [comment]
User figured it out on their own. Need to specify the tls.options label but with no value
